src/harness/fourslash.ts

```diff
@@ -2245,8 +2245,7 @@ namespace FourSlash {
 
         const diagnostics = ts.getPreEmitDiagnostics(program, sourceFile);
         if (diagnostics.length > 0) {
-            throw new Error(`Error compiling ${fileName}: ` +
-                diagnostics.map(e => ts.flattenDiagnosticMessageText(e.messageText, Harness.IO.newLine())).join("\r\n"));
+            throw new Error(`Error compiling ${fileName}: ${diagnostics.map(e => ts.flattenDiagnosticMessageText(e.messageText, Harness.IO.newLine())).join("\r\n")}`);

```

Comment:
Honestly I changed this just to get a 100% binary equivalence between compiling `jake tests` with transforms disabled and with it enabled.